Sources/FluentMongoDriver/SiblingsFieldProperty.swift

```diff
@@ -0,0 +1,171 @@
+import FluentKit
+
+extension Fields {
+    public typealias SiblingsField<To> = SiblingsFieldProperty<Self, To>
+        where To: Model
+}
+
+@propertyWrapper
+public final class SiblingsFieldProperty<From, To>
+    where From: Fields, To: Model
+{
+    public let key: FieldKey
+    public var value: [To]?
+    public var identifiers: [To.IDValue]?
+    
+    public var projectedValue: SiblingsFieldProperty<From, To> {
+        self
+    }
+
+    public var wrappedValue: [To] {
+        get {
+            guard let value = self.value else {
+                fatalError("Cannot access field before it is initialized or fetched: \(self.key)")
+            }
+            return value
+        }
+        set {
+            self.value = newValue
+        }
+    }
+    
+    public init(key: FieldKey) {
+        self.key = key
+    }
+
+    public func query(on database: Database) -> QueryBuilder<To> {
+        guard let identifiers = self.identifiers else {
+            fatalError("Cannot query siblings relation from unsaved model.")
+        }
+
+        return To.query(on: database)
+            .filter(\._$id ~~ identifiers)
+    }
+}
+
+extension SiblingsFieldProperty: PropertyProtocol {
+    public typealias Model = From
+    public typealias Value = [To]
+    
+}
+
+extension SiblingsFieldProperty: FieldProtocol { }
+extension SiblingsFieldProperty: AnyField { }
+
+extension SiblingsFieldProperty: Relation {
+    public var name: String {
+        return "SiblingsField<\(From.self), \(To.self)>(key: \(self.key))"
+    }
+
+    public func load(on database: Database) -> EventLoopFuture<Void> {
+        self.query(on: database).all().map {
+            self.value = $0
+        }
+    }
+}
+
+extension SiblingsFieldProperty: AnyProperty {
+    public var nested: [AnyProperty] { [] }
+    public var path: [FieldKey] { [self.key] }
+
+    public func input(to input: inout DatabaseInput) {
+        input.values[self.key] = identifiers.map { identifiers in
+            return .bind(identifiers)
+        }
+    }
+
+    public func output(from output: DatabaseOutput) throws {
+        if output.contains([self.key]) {
+            self.identifiers = nil
+            
+            do {
+                self.identifiers = try output.decode(self.key, as: [To.IDValue].self)
+            } catch {
+                throw FluentError.invalidField(
+                    name: self.key.description,
+                    valueType: [To.IDValue].self,
+                    error: error
+                )
+            }
+        }
+    }
+
+    public func encode(to encoder: Encoder) throws {
+        if let identifiers = self.identifiers {
+            var container = encoder.singleValueContainer()
+            try container.encode(identifiers)
+        }
+    }
+
+    public func decode(from decoder: Decoder) throws {
+        self.identifiers = try [To.IDValue](from: decoder)
```

Comment:
For cleanness-sake, maybe 
```suggestion
        self.identifiers = try .init(from: decoder)
```